lib/phoenix_live_view/channel.ex

```diff
@@ -339,10 +339,6 @@ defmodule Phoenix.LiveView.Channel do
     push(state, "live_redirect", %{to: to, kind: kind})
   end
 
-  defp push_internal_live_redirect(state, %{to: to, kind: kind}, ref) do
```

Comment:
So maybe we should not pass the `ref` at all? :)

Reply:
For me looks a good approach :+1:

Reply:
Sweet, let us know once you update it and we will merge it!

Reply:
Updated :smile: